bot/run.py

```diff
@@ -9,19 +11,29 @@
 
 
 bot = commands.Bot(
-    command_prefix=commands.when_mentioned_or(">"),
+    command_prefix=commands.when_mentioned_or('>'),
     description='Play audio from myinstants',
-    activity=Activity(type=ActivityType.listening, name='Discord removed my permissions, trying to fix :(')
+    activity=Activity(
+        type=ActivityType.listening,
+        name='Discord removed my permissions, trying to fix :(',
+    ),
+    intents=discord.Intents.all(),
```

Comment:
This will break the production bot since bots with more than 100 servers cannot use privileged intents.
Please use None or equivalent

Reply:
Changed it to default intents and added `message_content` as well, as it is necessary to read them